tests/actions/test.subs-actions.js

```diff
@@ -70,47 +70,34 @@ describe('subscription actions', function() {
 
   describe('createSubscription', function() {
 
+    var fakeFetch;
     var fakeCard = {
       number: '411111111111',
       cvv: '222',
       expiration: '06/16',
     };
 
-    // Bad card with missing cvv
-    // and expiration.
-    var fakeBadCard = {
-      number: '411111111111',
-    };
-
-    class FakeBraintreeClient {
-      tokenizeCard(config, callback) {
-        console.log('resolving braintree tokenization');
-        callback(null, 'some-nonce');
-      }
-    }
-
-    class FakeBraintreeClientError {
-      tokenizeCard(config, callback) {
-        console.log('resolving braintree tokenization with error');
-        callback("I'm some error");
-      }
-    }
+    beforeEach(function() {
+      dispatchSpy = sinon.spy();
+      fakeFetch = helpers.fakeFetch();
+    });
 
-    function createSubscription(fetch, client=FakeBraintreeClient,
-                                payMethod=fakeCard) {
-      var deferredAction = subActions.createSubscription(
-                                          'product-id',
-                                          payMethod,
-                                          'braintree-token',
-                                          fetch,
-                                          client);
+    function createSubscription({fetch=fakeFetch,
+                                 client=helpers.FakeBraintreeClient,
+                                 payMethod=fakeCard} = {}) {
```

Comment:
named parameters: http://www.2ality.com/2011/11/keyword-parameters.html

There's actually one thing you can do in JS that you can't do in Python! You can invoke the default by passing in undefined:
````
function foo({param='default-value'}) {
  console.log(param);
}
foo({param: undefined});  // prints 'default-value'
````
In python, you have to leave off the parameter altogether to accomplish this. It's not a big deal but sometimes it leads to silly code like this:

````
def foo(param='the-default'):
    print param

if thing:
    foo(param=thing)
else:
    foo()
````
or like this:

````
def foo(param=None):
    if not param:
        param = 'the-default'
    print param

foo(param=thing)
````